app/src/main/java/org/simple/clinic/scheduleappointment/patientFacilityTransfer/PatientFacilityChangeActivity.kt

```diff
@@ -21,8 +22,31 @@ class PatientFacilityChangeActivity : AppCompatActivity() {
   }
 
   override fun attachBaseContext(baseContext: Context) {
-    ClinicApp.appComponent.inject(this)
```

Comment:
If we remove this line, the class will break at runtime at this commit since `locale` would not have been initialized. This is fine for now since the activity is not being launched, though.

Reply:
Good catch!